src/gpgservice.py

```diff
@@ -0,0 +1,69 @@
+import gnupg
+import json
+import pprint
+import os
+
+class GPGService:
+
+	def __init__(self):
+		super(GPGService, self).__init__()
```

Comment:
What's the point of calling `super().__init__` in class that doesn't have any explicit parent? (honest question, not sarcasm)